Service does not start in production on v1.5.6

We recently tried to deploy [v1.5.6](https://github.com/OrcaCollective/OpenOversight/releases/tag/v1.5.6) into production and saw failures on startup with the following stacktrace:

```
web_1       | [2023-07-05 21:24:38 +0000] [13] [INFO] Worker exiting (pid: 13)
web_1       | /usr/local/lib/python3.11/site-packages/flask_limiter/extension.py:293: UserWarning: Using the in-memory storage for tracking rate limits as no storage was explicitly specified. This is not recommended for production use. See: https://flask-limiter.readthedocs.io#configuring-a-storage-backend for documentation about configuring the storage backend.
web_1       |   warnings.warn(
web_1       | [2023-07-05 21:24:38 +0000] [14] [ERROR] Exception in worker process
web_1       | Traceback (most recent call last):
web_1       |   File "/usr/local/lib/python3.11/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
web_1       |     worker.init_process()
web_1       |   File "/usr/local/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
web_1       |     self.load_wsgi()
web_1       |   File "/usr/local/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
web_1       |     self.wsgi = self.app.wsgi()
web_1       |                 ^^^^^^^^^^^^^^^
web_1       |   File "/usr/local/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
web_1       |     self.callable = self.load()
web_1       |                     ^^^^^^^^^^^
web_1       |   File "/usr/local/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
web_1       |     return self.load_wsgiapp()
web_1       |            ^^^^^^^^^^^^^^^^^^^
web_1       |   File "/usr/local/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
web_1       |     return util.import_app(self.app_uri)
web_1       |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web_1       |   File "/usr/local/lib/python3.11/site-packages/gunicorn/util.py", line 359, in import_app
web_1       |     mod = importlib.import_module(module)
web_1       |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web_1       |   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
web_1       |     return _bootstrap._gcd_import(name[level:], package, level)
web_1       |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web_1       |   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
web_1       |   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
web_1       |   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
web_1       |   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
web_1       |   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
web_1       |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
web_1       |   File "/usr/src/app/OpenOversight/app/__init__.py", line 153, in <module>
web_1       |     app = create_app()
web_1       |           ^^^^^^^^^^^^
web_1       |   File "/usr/src/app/OpenOversight/app/__init__.py", line 44, in create_app
web_1       |     limiter.init_app(app)
web_1       |   File "/usr/local/lib/python3.11/site-packages/flask_limiter/extension.py", line 302, in init_app
web_1       |     storage_from_string(
web_1       |   File "/usr/local/lib/python3.11/site-packages/limits/storage/__init__.py", line 63, in storage_from_string
web_1       |     return cast(StorageTypes, SCHEMES[scheme](storage_string, **options))
web_1       |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web_1       |   File "/usr/local/lib/python3.11/site-packages/limits/storage/memory.py", line 32, in __init__
web_1       |     self.timer.start()
web_1       |   File "/usr/local/lib/python3.11/threading.py", line 957, in start
web_1       |     _start_new_thread(self._bootstrap, ())
web_1       | RuntimeError: can't start new thread
web_1       | [2023-07-05 21:24:38 +0000] [14] [INFO] Worker exiting (pid: 14)
web_1       | [2023-07-05 21:24:39 +0000] [10] [WARNING] Worker with pid 13 was terminated due to signal 15
web_1       | [2023-07-05 21:24:39 +0000] [10] [WARNING] Worker with pid 12 was terminated due to signal 15
web_1       | [2023-07-05 21:24:39 +0000] [10] [WARNING] Worker with pid 14 was terminated due to signal 15
web_1       | [2023-07-05 21:24:39 +0000] [10] [INFO] Shutting down: Master
web_1       | [2023-07-05 21:24:39 +0000] [10] [INFO] Reason: Worker failed to boot.
```

I was able to roll back to [v1.5.5](https://github.com/OrcaCollective/OpenOversight/releases/tag/v1.5.5) and restart the service without issue. I tried to restart v1.5.6 several times but the same issue came up. Our production box has been up for 562 days and is currently at PID `4141152`, so perhaps that might be related? Although I'm not sure why we'd be able to run the old service if threads couldn't be started...

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.11.4-slim as base
+FROM python:3.11.4-slim-bullseye as base
 
 WORKDIR /usr/src/app
 

